web/src/components/Cart/Cart.js

```diff
@@ -94,12 +95,26 @@ const ShoppingCartButton = styled(Button)`
     `}
 `
 
-const CartItem = ({ image, quantity, name }) => {
+const RemoveFromCartButton = styled(Button)`
+  background-color: var(--red-7);
+
+  &:hover {
+    transition: background-color 200ms;
+  }
+`
```

Comment:
Got one more suggestion. Even after shrinking the icon, I feel like the button is still a little too big and red: 

<img width="219" alt="image" src="https://user-images.githubusercontent.com/32992335/166876175-1bb5d88c-ee48-41e3-bf34-1f1dfb0dc9d5.png">

There's an "icon" variant of the `Button` component; we already use it for the shopping cart and user portal. I kind of like how it looks here:

<img width="219" alt="image" src="https://user-images.githubusercontent.com/32992335/166876231-9f84b473-57e3-4860-8637-4fb51ed17574.png">

It's a destructive action, but it doesn't feel so destructive as to deserve the attention the size and color call to it.

Implementing this would mean getting rid of the `RemoveFromCartButton` and just making it just `Button variant="icon"`. What do you think?

Reply:
Yeah, that looks much better actually. I've updated my PR.

Let me know when then e2e issue is resolved, and I can likely dedicate a few hours to some other issues on this repo.